service/src/main/java/bio/terra/pipelines/stairway/imputation/SubmitCromwellRunSetStep.java

```diff
@@ -63,45 +66,179 @@ public StepResult doStep(FlightContext flightContext)
+                        new ParameterTypeDefinitionPrimitive()
+                            .primitiveType(PrimitiveParameterValueType.FILE)
+                            .type(ParameterTypeDefinition.TypeEnum.PRIMITIVE))
                     .destination(
                         new OutputDestinationRecordUpdate()
-                            .recordAttribute("output_file")
+                            .recordAttribute("chunks_info")
+                            .type(OutputDestination.TypeEnum.RECORD_UPDATE)))
+            // failed_chunks output
+            .addWorkflowOutputDefinitionsItem(
+                new WorkflowOutputDefinition()
+                    .outputName("%s.failed_chunks".formatted(wdlMethodName))
+                    .outputType(
+                        new ParameterTypeDefinitionPrimitive()
+                            .primitiveType(PrimitiveParameterValueType.FILE)
+                            .type(ParameterTypeDefinition.TypeEnum.PRIMITIVE))
+                    .destination(
+                        new OutputDestinationRecordUpdate()
+                            .recordAttribute("failed_chunks")
+                            .type(OutputDestination.TypeEnum.RECORD_UPDATE)))
+            // imputed_multi_sample_vcf output
+            .addWorkflowOutputDefinitionsItem(
+                new WorkflowOutputDefinition()
+                    .outputName("%s.imputed_multi_sample_vcf".formatted(wdlMethodName))
+                    .outputType(
+                        new ParameterTypeDefinitionPrimitive()
+                            .primitiveType(PrimitiveParameterValueType.FILE)
+                            .type(ParameterTypeDefinition.TypeEnum.PRIMITIVE))
+                    .destination(
+                        new OutputDestinationRecordUpdate()
+                            .recordAttribute("imputed_multi_sample_vcf")
+                            .type(OutputDestination.TypeEnum.RECORD_UPDATE)))
+            // imputed_multi_sample_vcf_index output
+            .addWorkflowOutputDefinitionsItem(
+                new WorkflowOutputDefinition()
+                    .outputName("%s.imputed_multi_sample_vcf_index".formatted(wdlMethodName))
+                    .outputType(
+                        new ParameterTypeDefinitionPrimitive()
+                            .primitiveType(PrimitiveParameterValueType.FILE)
+                            .type(ParameterTypeDefinition.TypeEnum.PRIMITIVE))
+                    .destination(
+                        new OutputDestinationRecordUpdate()
+                            .recordAttribute("imputed_multi_sample_vcf_index")
+                            .type(OutputDestination.TypeEnum.RECORD_UPDATE)))
+            // n_failed_chunks output
+            .addWorkflowOutputDefinitionsItem(
+                new WorkflowOutputDefinition()
+                    .outputName("%s.n_failed_chunks".formatted(wdlMethodName))
```

Comment:
just for niceness can you put this next to the other `failed_chunk` related variables